### DISCUSSION TITLE
How to visualize all commits? (reflog analog)

### DISCUSSION THREAD

**Discussion by ilyagarazha** [gitkraken/vscode-gitlens#2613] [Q&A] [answered]
https://github.com/mhutchie/vscode-git-graph/issues/89

Is there a way to visualize all commits in commit graph in the same way as in the vscode-git-graph extension ?

**Comment by jkelroy** [comment, marked as answer]
Hello @IlyaGarazha, not currently but there is an open feature request to support reflogs [here](https://github.com/gitkraken/vscode-gitlens/issues/1814).
